doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -1210,37 +1210,31 @@ NTLMSSP fields
 * "domain" (string): the Windows domain.
 * "user" (string): the user.
 * "host" (string): the host.
+* "version" (string): the version of client.
 
 Example::
 
-    "ntlmssp": {
-      "domain": "VNET3",
-      "user": "administrator",
-      "host": "BLU"
-    }
-
+  "ntlmssp": {
+      "domain": "CONTOSO",
+      "user": "Administrator",
+      "host": "SERVER01",
+      "version": "6.0 build 6001 rev 15"
+  }
```

Comment:
This CI error here: https://github.com/OISF/suricata/actions/runs/4073611297/jobs/7023535861#step:14:276

Is probably due to:
not having a new line between `} ` and `More complete example`.